website/docs/jellyfish/api/governance.md

```diff
@@ -0,0 +1,36 @@
+---
+id: governance
+title: Governance API
+sidebar_label: Governance API
+slug: /jellyfish/api/governance
+---
+
+```js
+import {Client} from '@defichain/jellyfish'
+const client = new Client()
+
+// Using client.governance.
+const something = await client.governance.method()
+```
+
+## createCfp
+
+Creates a Cummunity Fund Request.
+
+```ts title="client.governance.createCfp()"
+interface governance {
+  createCfp (data: CFPData, inputs: UTXO[]): Promise<string>
```

Comment:
Should be `createCfp (data: CFPData, inputs: UTXO[] = []): Promise<string> {` like what you wrote in the `governance.ts`